e2e/test/scenarios/actions/model-actions.cy.spec.js

```diff
@@ -847,6 +867,7 @@ function enableSharingFor(actionName, { publicUrlAlias }) {
   cy.findByRole("dialog").within(() => {
     cy.button("Action settings").click();
     cy.findByLabelText("Make public").should("not.be.checked").click();
+    cy.wait("@enableActionSharing");
```

Comment:
I think some failures were because sharing hadn't been properly enabled because we weren't ever waiting for the api request to complete before navigating to a new page.